File: Pandacap.LowLevel/ActivityPub/IActivityPubImage.fs
namespace Pandacap.ActivityPub

type IActivityPubImage =
    abstract member GetUrl: appInfo: ActivityPubHostInformation -> string
    abstract member AltText: string
    abstract member MediaType: string
    abstract member HorizontalFocalPoint: decimal option
    abstract member VerticalFocalPoint: decimal option

File: Pandacap.ActivityPub.Models.Interfaces/IActivityPubImage.cs
namespace Pandacap.ActivityPub.Models.Interfaces
{
    public interface IActivityPubImage
    {
        string Url { get; }
        string AltText { get; }
        string MediaType { get; }
        decimal? HorizontalFocalPoint { get; }
        decimal? VerticalFocalPoint { get; }
    }
}
